TheForceEngine/TFE_DarkForces/Actor/actorModule.h

```diff
@@ -30,13 +30,20 @@ enum LogicAnimFlags
 
 enum LogicAnimState : u32
 {
+	//GENERAL
 	STATE_DELAY = 0u,
+	//ATTACK MODULE
 	STATE_ANIMATEATTACK,
 	STATE_FIRE1,
 	STATE_ANIMATE1,
 	STATE_FIRE2,
 	STATE_ANIMATE2,
-	STATE_COUNT
+	STATE_COUNT,
+	//THINKER MODULE
+	// Move towards our destination
+	STATE_MOVE = 1u,
+	// Update our destination (generally moving towards the player).
+	STATE_TURN = 2u,
 };
```

Comment:
@kevinfoley My opinion is that it would be cleaner, and more readable code if we just make a separate enum

So change the existing `LogicAnimState` to `LogicAttackAnimState` (or something like that)

and add a `LogicThinkerAnimState` with MOVE and TURN

Reply:
as discussed in Discord - this won't work because both modules use the same struct